gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Client.java

```diff
@@ -914,6 +922,26 @@ public Settings create() {
         }
     }
 
+    public static class ClientUtil {
```

Comment:
Why did you make a class to hold this static method?

Reply:
Moved method to RequestOptions